api/applications_api/filters.py

```diff
@@ -8,3 +8,13 @@ class ApplicationFilter(filters.FilterSet):
         field_name="application_round", queryset=ApplicationRound.objects.all()
     )
     status = filters.BaseInFilter(field_name="cached_latest_status")
+
+
+class ApplicationEventWeeklyAmountReductionFilter(filters.FilterSet):
+    application_event_id = filters.NumberFilter(method="filter_application_event")
```

Comment:
`filter.ModelChoiceFilter` would be nice built-in filter to use here.  If you use it, the method below can be removed.

Reply:
I know there's some other filtering made with NumberFilter / method (by me) => those should be refactored at some point :hourglass:

Reply:
Yeah, i'll change this